decidim/decidim
# Create taxonomies section in the general settings

#### :tophat: What? Why?
This pull request introduces a unified section for managing different taxonomies (territorial scopes, thematic areas, etc.) in the general configuration.

#### :pushpin: Related Issues
- Fixes #12997

#### Testing
1. As an admin, access the general settings and check for the new taxonomies section.
2. Verify that the taxonomies section displays a table with columns: drag and drop action, Name, Amount, Actions (edit and delete).
3. Ensure the search option is available on top of the table.
4. Test creating a new taxonomy and verify the ability to nest items.
5. Ensure items can be sorted using drag and drop functionality for both parent and child items.
6. Create, edit, and delete taxonomy items and verify that the interface is intuitive and user-friendly.

### :camera: Screenshots
![Screenshot 2024-07-17 at 22 38 34](https://github.com/user-attachments/assets/a5aa82d0-29dc-45c2-9195-f6e656e7b57d)

:hearts: Thank you!
